packages/common/src/stopwatch.ts

```diff
@@ -0,0 +1,44 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+import updateLocale from "dayjs/plugin/updateLocale";
+
+dayjs.extend(relativeTime);
+dayjs.extend(updateLocale);
+dayjs.updateLocale("en", {
```

Comment:
Why you updated the locale? Did it not work with dayjs.locale("en")?

Reply:
No, because I needed to overwrite the seconds formater to not display "a few seconds" but rather "2 seconds"